### PR TITLE
ci: upload artifacts

### PR DESCRIPTION
**Pull request by ahaoboy**
### Description of changes

For the convenience of testing, upload the artifacts

### CODE REVIEW

**Comment by ahaoboy** [comment]
https://github.com/awslabs/llrt/actions/runs/10925799624/job/30328199206

```
failures:

---- modules::net::socket::tests::test_server_echo stdout ----
thread 'modules::net::socket::tests::test_server_echo' panicked at llrt_modules\src\modules\net\socket.rs:430:70:
called `Result::unwrap()` on an `Err` value: Os { code: 10013, kind: PermissionDenied, message: "An attempt was made to access a socket in a way forbidden by its access permissions." }
stack backtrace:
   0: rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::result::unwrap_failed
   3: <tokio::future::maybe_done::MaybeDone<Fut> as core::future::future::Future>::poll
   4: <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
   5: llrt_modules::modules::net::socket::tests::test_server_echo::{{closure}}::{{closure}}::{{closure}}
   6: llrt_modules::test::test_async_with::{{closure}}::{{closure}}::{{closure}}
   7: <rquickjs_core::context::async::future::WithFuture<F,R> as core::future::future::Future>::poll
   8: llrt_modules::modules::net::socket::tests::test_server_echo::{{closure}}
   9: tokio::runtime::scheduler::current_thread::Context::enter
  10: tokio::runtime::context::scoped::Scoped<T>::set
  11: tokio::runtime::scheduler::current_thread::CoreGuard::block_on
  12: tokio::runtime::context::runtime::enter_runtime
  13: tokio::runtime::runtime::Runtime::block_on
  14: llrt_modules::modules::net::socket::tests::test_server_echo
  15: core::ops::function::FnOnce::call_once
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by nabetti1720** [comment]
Hi @ahaoboy ,  The error I am getting in the Build Windows binaries section is caused by my previous PR. Specifically, when I refactored the makefile, I made an error in specifying the target file name for the zip.

Can you please correct this in this PR?

```
zip -j llrt-windows-x64.zip target/x86_64-pc-windows-gnu/release/llrt/release/llrt.exe
	zip warning: name not matched: target/x86_64-pc-windows-gnu/release/llrt/release/llrt.exe

zip error: Nothing to do! (llrt-windows-x64.zip)
make: *** [Makefile:101: llrt-windows-x64.zip] Error 12
```

https://github.com/awslabs/llrt/blob/62b60c9d65a2e351facfa859e7b33915e238e12c/Makefile#L96

The correct information is as follows.

```makefile
// makefile
zip -j $$@ target/$$(TARGET_windows_$$(RELEASE_ARCH_NAME_${1}))/release/llrt.exe
```

**Comment by ahaoboy** [comment]
> The correct information is as follows.
> 
> ```makefile
> // makefile
> zip -j $$@ target/$$(TARGET_windows_$$(RELEASE_ARCH_NAME_${1}))/release/llrt.exe
> ```


Thanks for your help

**Comment by ahaoboy** [comment]
https://github.com/ahaoboy/llrt/actions/runs/10937408801

I've tried using the msvc toolchain on the Windows platform and it seems to work too, though it's a little bit larger and there may be a timing inconsistency issue that I hesitate to add to ci

File: llrt_core/build.rs [hunk 1]
```diff
@@ -265,7 +265,20 @@ fn generate_compression_dictionary(
...
+        .iter()
+        .map(|i| {
+            Path::new(i)
+                .strip_prefix(out_dir.clone())
+                .unwrap()
+                .to_string_lossy()
+                .to_string()
+        })
+        .collect();
+    let mut cmd = cmd.current_dir(out_dir).args(short_source_files).spawn()?;
```
Comment (richarddavison) [inline llrt_core/build.rs:281]: Nice!

**Review by richarddavison — APPROVED** [review]

**Comment by richarddavison** [comment]
Thanks LGTM!

**Pull request merged**